samples/transient-clients/sample.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Transient Client Sample
+reviewed: 2017-05-16
+component: Core
+related: 
+ - nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed
+ - samples/show-case
+---
+
+This sample demonstrates occasionally connected clients that only need to received event updates while connected. In this case it is best not to connect the clients directly to the bus, but instead use a push technology, such as [SignalR](http://signalr.net/), to update clients while they are connected.
```

Comment:
> need to received event updates

receive

> In this case it is best not to connect the clients directly to the bus

Since we've walked away from using "the bus" would it be better to say "In this case it is best not to implement each client as a full NServiceBus endpoint"

Should there be a paragraph bullet list of the whys? Clients may go dark for a long time, laptop gets disconnected from a network, etc. Messages start piling up and can threaten the rest of the system, but client probably doesn't care about event after X time anyway.